app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post, foreign_key: 'post_id'
+
+  def increment_comment_counter
+    puts 'Incrementing comment counter'
+    user.increment!(:comments_counter)
```

Comment:
- [ ] Please increment the comments_counter for the post, not the user as the comments should belong to a post.